src/uwp/Microsoft.Build.Net.CoreRuntimeTask/ResourceHandlingTask.cs

```diff
@@ -66,21 +76,37 @@ public bool IsUpToDate(string assemblyPath, out string reswFilePath)
                         _logger.LogMessage(MessageImportance.Low, Resources.Message_CachedReswNotUpToDateAssemblyNewer, assemblyPath); 
                         return false;
                     }
-                    if (info.ResWPath == null || !File.Exists(info.ResWPath))
+                    if (info.ResWInfoList == null)
                     {
-                        _logger.LogMessage(MessageImportance.Low, Resources.Message_CachedReswNotExists, assemblyPath, info.ResWPath); 
```

Comment:
>_logger.LogMessage(M [](start = 24, length = 20)

why removed the logging here? we can just update it.

Reply:
This logging is kept in the foreach loop below.